files/configfile/postfix_conf.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+#
+# This script gets PostFix ready to support the SMTP needs of
+# a PWM service deployed to AWS.
+#
+#################################################################
+
+# Compute networks to give SMTP access to
+MYIPS=($(ifconfig -a | awk '/inet /{ print $2 }'))
+MYNETS=(127.0.0.0/8)
+for IP in ${MYIPS[@]}
+do
+  MYNETS+=($(ip route show | awk '/'${IP}'/{print $1}'))
+done
+
+MDURI="http://169.254.169.254"
+MDPATH="${MDURI}/latest/dynamic/instance-identity/document/"
+AWSREG=$(curl -s ${MDPATH} | awk '/"region"/{print $3}' | \
+  sed -e 's/"//g' -e 's/,//g')
+THISHOST=$(hostname)
+THISDOM=$(hostname | sed 's/'$(hostname -s)'\.//')
+SMTPVERS=$(rpm --qf '%{version}' -q postfix)
+
+# Update PostFix's primary configuration file
+mv /etc/postfix/main.cf /etc/postfix/main.cf-BAK && \
```

Comment:
This file should really be managed by salt via `file.managed`

Reply:
Not really sure what you are looking for on this one. The configfile.sls is managing this with file.managed

get_postfix_config_script:
  file.managed:
    - name: /usr/local/bin/postfix_conf.sh
    - source: salt://files/configfile/postfix_conf.sh
    - template: jinja
    - mode: 700

Reply:
Done